core/src/test/scala/kanaloa/reactive/dispatcher/queue/QueueSpec.scala

```diff
@@ -248,7 +247,7 @@ class QueueScope(implicit system: ActorSystem) extends ScopeWithQueue {
   def waitForWorkerRegistration(queue: QueueRef, numberOfWorkers: Int): Unit = {
```

Comment:
aside from this function not working(we removed the `Queue`'s `QueryStatus` handler, this function is also not used, so we should remove it.